spec/factories/run_factory.rb

```diff
@@ -16,6 +16,9 @@
     # a bad run and a better run from the same category, against which suggestions can be generated
     compare_subject:          {filename: 'compare_subject.lss'},
     compare_object:           {filename: 'compare_object.lss'},
+
+    # specific-content runs
+    with_segments_bests:       {filename: 'with_segments_bests.lss'}
```

Comment:
I took the file from the actual bug report, not sure if that's an overkill. I think of this in the context of the factory as a file that represent a specific scenario to test, maybe in the future some more will show up, but let me know your thoughts.